ansible/cloud_providers/azure_infrastructure_deployment.yml

```diff
@@ -8,16 +8,17 @@
     - step001
     - deploy_infrastructure
   environment:
+    # These first four are only used by Ansible modules
     AZURE_CLIENT_ID: "{{azure_service_principal}}"
     AZURE_TENANT: "{{azure_tenant}}"
     AZURE_SECRET: "{{azure_password}}"
     AZURE_SUBSCRIPTION_ID: "{{azure_subscription_id}}"
     # AZURE_CONFIG_DIR: create a specific config dir for this stack to allow concurrent access
+    # This variable is also used by the CLI
     AZURE_CONFIG_DIR: "/tmp/.azure-{{project_tag}}"
+    PATH: "/tmp/.azure-{{project_tag}}:/usr/local/bin:/usr/bin"
```

Comment:
Can  you use `output_dir` for that instead ?

With output_dir, our script always make sure everything is cleaned up from the deployer host, whatever happens to the process.

Whereas, if you use something in /tmp/, it's not guaranted.

If it's a secret, you can put it in `{{ output_dir }}/secrets`, it will not be save in the tarball (used for troubleshooting sometimes)

Reply:
Sure, I'll fix this so it uses output_dir.

Reply:
Its not the output_dir, output_dir doesn't seem to be defined in the configs I was working with. (aro and ocp4-cluster).

Is output_dir defined every time by default through RHPDS like guid?

I'm ok with making this a variable instead of hardcoding /tmp/, but I want to make sure I'm using something that will exist every time.

Reply:
If it's not defined in the config, it should.
`output_dir` should be always defined.
Yes, please don't use `/tmp/` as it could fill up the `/tmp/` of the deploy host.

@rhjcd  FYI. 
Why did this PR was merged before this was resolved ?